irc.go

```diff
@@ -596,15 +603,15 @@ func (irc *Connection) negotiateCaps() error {
 		remaining_caps--
 	}
 
-	irc.pwrite <- fmt.Sprintf("CAP END\r\n")
+	irc.pwrite <- "CAP END\r\n"
 
 	return nil
 }
 
 // Create a connection with the (publicly visible) nickname and username.
 // The nickname is later used to address the user. Returns nil if nick
 // or user are empty.
-func IRC(nick, user string) *Connection {
+func IRC(nick, user string, myhost string) *Connection {
```

Comment:
This breaks backwards compatibility.

Reply:
Sorry I add this functionality for my own project and im not pro in go, I try to find a better solution for this future. 
Maybe you can suggest the way?